torch/_refs/__init__.py

```diff
@@ -646,6 +646,14 @@ def logsumexp(
     return result
 
 
+@register_decomposition(torch.ops.aten.special_multigammaln)
+@out_wrapper()
+def multigammaln(a: TensorLikeType, dim: int) -> TensorLikeType:
+    c = 0.25 * dim * (dim - 1) * prims.log(torch.pi)
+    b = prims.arange(0, dim) / 2
+    return sum(prims.lgamma(a - b), -1) + c
```

Comment:
I think the implementation should be moved to https://github.com/pytorch/pytorch/blob/master/torch/_refs/special/__init__.py

Reply:
also, call `torch.lgamma`, `torch.log`, etc. These call will then be transformed into refs / prims as needed by a context manager.